src/Module/Hooks/DisplayAdminProductsExtra.php

```diff
@@ -5,22 +5,28 @@
 use Configuration;
 use Db;
 use Gett\MyparcelBE\Constant;
+use Gett\MyparcelBE\Database\Table;
 
 trait DisplayAdminProductsExtra
 {
+    /**
+     * @throws \PrestaShopDatabaseException
+     */
     public function hookActionProductUpdate(array $params): void
     {
-        Db::getInstance()->delete(
-            'myparcelbe_product_configuration',
-            'id_product = ' . (int) $params['id_product']
-        );
+        Db::getInstance()
+            ->delete(
+                Table::TABLE_PRODUCT_CONFIGURATION,
+                'id_product = ' . (int) $params['id_product']
+            );
         foreach ($_POST as $key => $item) {
-            if (stripos($key, $this->name) === 0) {
-                Db::getInstance()->insert('myparcelbe_product_configuration', [
-                    'id_product' => (int) $params['id_product'],
-                    'name' => $key,
-                    'value' => $item,
-                ]);
+            if (0 === stripos($key, $this->name)) {
+                Db::getInstance()
+                    ->insert(Table::TABLE_PRODUCT_CONFIGURATION, [
+                        'id_product' => (int) $params['id_product'],
+                        'name'       => $key,
```

Comment:
This looks like a security issue, unless the insert escapes the values.

Reply:
Was al zo, maar genoteerd